modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1277,6 +1287,24 @@ private List<ObjectDefinition> _getObjectDefinitions() {
 		return objectDefinitions;
 	}
 
+	private long _getObjectFolderId(long companyId, long objectFolderId)
+		throws PortalException {
+
+		if (objectFolderId == 0) {
+			ObjectFolder objectFolder =
+				_objectFolderLocalService.
+					getObjectFolderByExternalReferenceCode(
+						"uncategorized", companyId);
+
+			objectFolderId = objectFolder.getObjectFolderId();
+		}
+		else {
+			_objectFolderLocalService.getObjectFolder(objectFolderId);
+		}
+
+		return objectFolderId;
```

Comment:
I think we could improve this method :thinking: 
Like:
- we could add the return in the line 1299
- I think we could remove ELSE condition since we have the return inside the IF, and just make this call "getObjectFolder"; after this, keep the return as it is

What do you think?

Reply:
Ok, sounds good for me.